src/plugins/ffe/ffe.py

```diff
@@ -106,7 +106,7 @@ def migration_manager(self) -> PluginMigrationManager:
 
     @hookimpl
     def on_init(self):
-        if not FfeDatabase().check():
+        if FfeDatabase().check() == False:
```

Comment:
Don't check boolean equality with `== False`, check with their truthy value
```suggestion
        if not FfeDatabase().check():
```